src/Io/CompositeConnection.php

```diff
@@ -11,6 +11,7 @@
 /** @internal */
 class CompositeConnection extends EventEmitter implements ConnectionInterface
 {
+    private $remote;
```

Comment:
@yadaiio Good catch. This doesn't appear to be used anywhere except for the single `null` assignment, perhaps we should rather remove this entirely? /cc @SimonFrings

Reply:
Thank you @clue! After discussing this topic with @SimonFrings we decided to remove this as you said. 👍